www/src/utils/node/blog.js

```diff
@@ -82,7 +82,7 @@ exports.createPages = async ({ graphql, actions }) => {
         limit: 10000
         filter: {
           fileAbsolutePath: { ne: null }
-          frontmatter: { draft: { eq: false } }
+          frontmatter: { draft: { ne: true } }
```

Comment:
should this not be the same?
🤔 

or is there some case with `undefined`?

Reply:
correct. for articles that aren't drafts, `draft` is undefined, and so checking for `draft === false` will cause everything to fail.

Reply:
That was what was causing my errors from earlier, because none of the non-draft blog posts were being considered.